pkg/postage/service_test.go

```diff
@@ -99,24 +102,37 @@ func TestGetStampIssuer(t *testing.T) {
 	}
 	t.Run("found", func(t *testing.T) {
 		for _, id := range ids[1:4] {
-			st, err := ps.GetStampIssuer(id)
+			st, save, err := ps.GetStampIssuer(id)
 			if err != nil {
 				t.Fatalf("expected no error, got %v", err)
 			}
+			_ = save()
 			if st.Label() != string(id) {
 				t.Fatalf("wrong issuer returned")
 			}
 		}
+
+		// check if the save() call persisted the stamp issuers
+		for i, id := range ids[1:4] {
+			issuer := new(postage.StampIssuer)
+			err := store.Get(fmt.Sprintf("postage%d%d", chainID, i), issuer)
+			if err != nil {
+				t.Fatal(err)
+			}
+			if !bytes.Equal(id, issuer.ID()) {
```

Comment:
Since those byte slices are quite long and difficult to compare by eye in case of a mismatch, I recommend doing the following:
```Golang
if diff := cmp.Diff(id, issuer.ID()); diff != "" {
	t.Fatalf("issuer mismatch (-want +have):\n%s", diff)
}
```